src/content/ui-api/observability/prometheus/disable-monitoring/index.md

```diff
@@ -0,0 +1,80 @@
+---
+linkTitle: Disable Monitoring
+title: Disable Monitoring
+description: A guide explaining how to disable cluster monitoring
+weight: 50
+menu:
+  main:
+    identifier: uiapi-observability-clustermonitoring
+    parent: uiapi-observability-prometheus
+user_questions:
+  - How can I disable monitoring on my cluster ?
+aliases:
+  - /observability/prometheus/disable-monitoring
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+last_review_date: 2022-11-21
+---
+
+In this document you will learn how you can disable monitoring for you cluster.
+
+## Introduction to monitoring
+
+Each cluster created on the Giantswarm Platform benefits from our monitoring which allow us to provide you with 24/7 support to ensure best quality of service.
```

Comment:
```suggestion
Each cluster created on the Giant Swarm Platform benefits from our monitoring which allow us to provide you with 24/7 support to ensure best quality of service.
```

@marians do we like 'Giant Swarm Platform'? do we use 'Giant Swarm' just generally in these kinds of cases?